libraries/AC_AutoTune/AC_AutoTune.h

```diff
@@ -16,14 +16,53 @@
   support for autotune of multirotors. Based on original autotune code from ArduCopter, written by Leonard Hall
   Converted to a library by Andrew Tridgell
  */
-
 #pragma once
 
 #include <AP_HAL/AP_HAL.h>
-#include <AC_AttitudeControl/AC_AttitudeControl_Multi.h>
+#include <AC_AttitudeControl/AC_AttitudeControl.h>
 #include <AC_AttitudeControl/AC_PosControl.h>
+#include <AP_Math/AP_Math.h>
+
+#define AUTOTUNE_AXIS_BITMASK_ROLL            1
+#define AUTOTUNE_AXIS_BITMASK_PITCH           2
+#define AUTOTUNE_AXIS_BITMASK_YAW             4
+
+#define AUTOTUNE_PILOT_OVERRIDE_TIMEOUT_MS  500     // restart tuning if pilot has left sticks in middle for 2 seconds
```

Comment:
I guess not all of these need to move to the .h file.  It's perhaps a little messy to have some in the .h (so they can be shared with the child classes) and some in the .cpp if they are only used in the parent but it seems like we already have a list of definitions in both places anyway.

These can be left in AC_AutoTune.cpp:

- AUTOTUNE_PILOT_OVERRIDE_TIMEOUT_MS
- AUTOTUNE_LEVEL_ANGLE_CD
- AUTOTUNE_LEVEL_RATE_RP_CD
- AUTOTUNE_LEVEL_RATE_Y_CD
- AUTOTUNE_REQUIRED_LEVEL_TIME_MS
- AUTOTUNE_LEVEL_TIMEOUT_MS
- AUTOTUNE_TARGET_ANGLE_RLLPIT_CD
- AUTOTUNE_TARGET_RATE_RLLPIT_CDS
- AUTOTUNE_TARGET_MIN_ANGLE_RLLPIT_CD
- AUTOTUNE_TARGET_MIN_RATE_RLLPIT_CDS
- AUTOTUNE_TARGET_ANGLE_YAW_CD
- AUTOTUNE_TARGET_RATE_YAW_CDS
- AUTOTUNE_TARGET_MIN_ANGLE_YAW_CD
- AUTOTUNE_TARGET_MIN_RATE_YAW_CDS
- AUTOTUNE_MESSAGE_xxxx

the AUTOTUNE_DWELL_CYCLES definition can be removed it seems although perhaps it is required when tradheli is added although then it should probably be in the AC_AutoTune_Heli.cpp.

Reply:
I had a lot of issues with leaving these defines in .cpp because these defines are used in the child classes and I get compile errors if I put them in .cpp.

Reply:
done.  Thanks!